ariadne/contrib/federation/schema.py

```diff
@@ -26,26 +26,48 @@
         _service: _Service!
     }}
 
-    directive @external on FIELD_DEFINITION
-	directive @requires(fields: String!) on FIELD_DEFINITION
-	directive @provides(fields: String!) on FIELD_DEFINITION
-	directive @extends on OBJECT | INTERFACE
+    directive @extends on OBJECT | INTERFACE
+    directive @external on OBJECT | FIELD_DEFINITION
```

Comment:
`@external` was only applicable on `FIELD_DEFINITION` in v1, v2 added support for `OBJECT`